### ISSUE TITLE
Use new properties for evidence

### ISSUE THREAD

**Issue by hdietze** [geneontology/minerva#13]
We need to update the default properties for evidence handling:
 * AnnotationProperty for evidence is: `RO:0002162` "axiom has evidence"
 * ObjectProperty for linking a publication: `IAO:0000136`
 * Type for PMID instances: `IAO:0000311` Publication

The requires that IAO is added to the import chain. Also verify first the RO:0002162 is in the current tbox.

**Comment by hdietze** [comment]
Coordinate with @jnguyenx and @kltm synchronized changes (JS libraries)

**Comment by kltm** [comment]
I'll be folding these changes in as I work on geneontology/noctua#141 .

**Comment by hdietze** [comment]
Problem: `RO:0002162` is **not** `axiom has evidence`

**Comment by hdietze** [comment]
The correct relationship id is:
`http://purl.obolibrary.org/obo/RO_0002612` or `RO:0002612`
Please note the reversal of 6 and 1 here.

**Comment by hdietze** [comment]
Also we need to decide on the relation (and type) for the `with` instances. At the moment this is modeled as a string literal in the evidence object

**Comment by cmungall** [comment]
Note that is-about goes from publication to evidence. We can define an inverse if preferred

**Comment by cmungall** [comment]
Documentation here:

https://github.com/geneontology/minerva/blob/master/specs/owl-model.md#axiom-annotations-and-evidence

![diagram](https://github.com/geneontology/minerva/blob/master/specs/evidence-model.png)

**Comment by kltm** [comment]
@cmungall you might want to ditch the old diagram and/or add the new one. Or a ref to the most recent.

**Comment by kltm** [comment]
Clarification here about the virtual edge's "axiom has evidence": that's just what we're pretending in our heads right? In reality (i.e. file, client model) we're just pointing to an individual by id reference...

**Comment by cmungall** [comment]
Correct. An OWL annotation value cannot be an Object, it can only be the IRI of the object. I debated whether to explicitly represent this. When the diagram is presented alongside the example OWL this should hopefully be clearer: https://github.com/geneontology/minerva/blob/master/specs/owl-model.md#axiom-annotations-and-evidence

This distinction (IRI != Object) only happens at the OWL interpretation level. The diagram is a precise representation of the RDF, where IRI == Object, and the RO:0002162 triple points directly at the evidence individual.

> is that still in the cards?
not sure I understand the question

**Comment by cmungall** [comment]
This ticket could be split

 - switching properties is trivial, the models will be structurally isomorphic on the RDF level
 - switching modeling PMIDs from literals to individuals changes things structurally and has potential consequences for the UI, since individuals are shown as nodes on the display
